src/data_collectors/dummy_dynamics_data_collector.py

```diff
@@ -0,0 +1,25 @@
+from typing import Any
+
+import torch
+from torch import Tensor
+
+from ..utils.step_record import RecordKeys as RK
+from .data_collector import DataCollector
+
+
+class DummyDynamicsDataCollector(DataCollector):
+    def __init__(self, action_dim, observation_dim):
```

Comment:
- 引数はshapeにしておくと汎用性が出ます。観測は特に画像ですし。
- コンストラクタの引数に、`get_data`で返すデータ数を指定してください。また、`collect`でbatchを増やすのはダミークラスの機能として冗長かつ明示性が失われるため空の関数に置き換えてください。


- 引数の型アノテーションは全てにおいて忘れないでください。入出力の関係は他の処理に影響しますから、必ず明示する必要があります。
```suggestion
    def __init__(self, size: int,  action_shape: tuple[int], observation_shape: tuple[int]):
```

Reply:
修正しました https://github.com/MLShukai/PrimitiveAMI/pull/53/commits/b2fa5014a6e406b45d97bd2d2e8041f02c7730fb

Reply:
@myxyy 引数を`*_dim`から`*_shape`に変更するの忘れてますよ